python/sources/InfluxDB/library.json

```diff
@@ -0,0 +1,75 @@
+{
+  "libraryItemId": "3f9f4c8d-9590-456b-a7c1-89daa1155a6c",
+  "name": "InfluxDB 3.0",
+  "language": "Python",
+  "tags": {
+    "Complexity": ["Easy"],
+    "Technology": ["QuixStreams"],
+    "Pipeline Stage": ["Source"],
+    "Type": ["Connectors"],
+    "Vendor": ["InfluxData"]
+  },
+  "shortDescription": "Read from InfluxDB 3.0 and publish to Quix streams",
+  "DefaultFile": "main.py",
+  "EntryPoint": "build/dockerfile",
+  "RunEntryPoint": "main.py",
+  "IconFile": "icon.png",
+  "Variables": [
+    {
+      "Name" : "output",
+      "Type" : "EnvironmentVariable",
+      "InputType" : "OutputTopic",
+      "Description" : "This is the Quix topic that will receive the stream",
+      "DefaultValue" : "influxdb",
+      "Required": true
+  },
+    {
+      "Name": "task_interval",
+      "Type": "EnvironmentVariable",
+      "InputType": "FreeText",
+      "Description": "Interval to run query. Must be within the InfluxDB notation; 1s, 1m, 1h, 1d, 1w, 1mo, 1y",
+      "DefaultValue": "5m",
+      "Required": true
+    },
+    {
+      "Name": "INFLUXDB_HOST",
+      "Type": "EnvironmentVariable",
+      "InputType": "FreeText",
+      "Description": "Host address for the InfluxDB instance.",
+      "DefaultValue": "eu-central-1-1.aws.cloud2.influxdata.com",
+      "Required": true
+    },
+    {
+      "Name": "INFLUXDB_TOKEN",
+      "Type": "EnvironmentVariable",
+      "InputType": "FreeText",
+      "Description": "Authentication token to access InfluxDB.",
+      "DefaultValue": "<TOKEN>",
+      "Required": true
+    },
+    {
+      "Name": "INFLUXDB_ORG",
+      "Type": "EnvironmentVariable",
+      "InputType": "FreeText",
+      "Description": "Organization name in InfluxDB.",
+      "DefaultValue": "<ORG>",
+      "Required": false
```

Comment:
I feel like this should be mandatory. But if it's really optional then pls ignore this comment.

Reply:
I might remove this ORG environment variable altogether as with 3.0 you do not need it to query only to write.